### DISCUSSION TITLE
What is the difference between watchers_count and watchers in the response of GET /user/repos?

### DISCUSSION THREAD

**Discussion by termjobTest** [community/community#24500] [Apps, API and Webhooks]
<p>I have used <code>GET /user/repos</code>  for getting the repository details of an authenticate use, But in the response, I have two attributes <strong>watchers</strong> and <strong>watchers_count</strong>, in this case, which one I have to use for getting the watchers of the corresponding repository?</p>

**Comment by francisfuzz** [comment, marked as answer]
<aside class="quote no-group" data-username="termjobTest" data-post="1" data-topic="123484">
<div class="title">
<div class="quote-controls"></div>
 termjobTest:</div>
<blockquote>
<p>in this case, which one I have to use for getting the watchers of the corresponding repository?</p>
</blockquote>
</aside>
<p>:wave: <a class="mention" href="/termjobtest">@termjobTest</a> -  Because those fields are equivalent, you can use either for your use case.</p>

**Comment by termjobTest** [comment]
<p>Thank you <a class="mention" href="/francisfuzz">@francisfuzz</a></p>
